natspec/l2-token/IBurnMintERC20.md

```
@@ -0,0 +1,253 @@
+
+Burns tokens from the sender.
+
+*this function decreases the total supply.*
+
+#### Parameters
+
+| Name | Type | Description |
+|---|---|---|
+| amount | uint256 | The number of tokens to be burned. |
+
+### burn
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)